packages/compass-global-writes/src/store/reducer.ts

```diff
@@ -1,8 +1,14 @@
 import type { Action, Reducer } from 'redux';
 import type { GlobalWritesThunkAction } from '.';
-import { openToast } from '@mongodb-js/compass-components';
+import {
+  openToast,
+  showConfirmation as showConfirmationModal,
+} from '@mongodb-js/compass-components';
 import type { ManagedNamespace } from '../services/atlas-global-writes-service';
 
+const POLLING_INTERVAL = 5000;
+export const TEST_POLLING_INTERVAL = 100;
```

Comment:
Let's try to avoid this test-only stuff, we have options: you can make the interval configurable through the component props if really needed, as we are still just testing stores here without the UI you can use sinon to mock these timers, you can also just emit the same events that would happen on your timeout to simulate whatever you need to happen without the need to wait for actual timeout to hit

Reply:
thanks! went with sinon fake timers